releases/3.3.0/artifacts.json

```diff
@@ -0,0 +1,242 @@
+{
+  "3.3.0": {
+    "holoscan": {
+      "debian-version": "3.3.0-1",
+      "wheel-version": "3.3.0",
+      "base-images": {
+        "dgpu": "nvcr.io/nvidia/cuda:12.8.1-runtime-ubuntu24.04",
+        "igpu": "nvcr.io/nvidia/tensorrt:25.03-py3-igpu"
+      },
+      "build-images": {
+        "igpu": {
+          "jetson-agx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu",
+          "igx-orin-devkit": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu",
+          "sbsa": "nvcr.io/nvstaging/holoscan/holoscan:v3.3.0.0-igpu"
```

Comment:
Igpu+sbsa is not a valid combination

Reply:
Thank you @agirault. I have removed that option in #40.